src/checkbox/styles/checkbox.css

```diff
@@ -0,0 +1,78 @@
+@import '../../common/styles/variables.css';
+
+.label {
+	display: block;
+	font: var(--base-font);
+	margin-bottom: var(--line-spacing);
+	padding: 0.25em 0 0.25em 2em;
+	position: relative;
+}
+
+.root {
+	opacity: 0;
+	position: absolute;
+	z-index: -1;
+}
+
+.inputWrapper {
+	left: 0;
+	position: absolute;
+	top: 0;
+}
+
+.inputWrapper::before,
+.inputWrapper::after {
+	box-sizing: border-box;
+	content: '';
+	display: block;
+}
+
+.inputWrapper::before {
+	border: 2px solid var(--form-highlight-color);
+	height: 1.5em;
+	width: 1.5em;
+}
+
+.inputWrapper::after {
+	background-color: var(--form-highlight-color);
+	height: calc(1.5em - 6px);
```

Comment:
Yeah, I think ideally this will get split out to a theme and probably changed. I've just been writing more than basic styles to show that you can with this widget structure.